### ISSUE TITLE
Sniper CLONE not using AC_DOUBLE ? double strafe

### ISSUE THREAD

**Issue by krispaul27** [rathena/rathena#7689]
### rAthena Hash

5931

### Client Date

2015-11-04

### Server Mode

Pre-Renewal

### Result

I Just Trying to splay with clone skill, 
The clone used skill but it does not use AC_DOUBLE or 

i think this code means alot?

### Relevant Log Output

```Shell
/**
			 * The clone should be able to cast the skill (e.g. have the required weapon) bugreport:5299)
			 **/
			if( !skill_check_condition_castbegin(sd,skill_id,sd->status.skill[sk_idx].lv) )
				continue;
```

### Expected Result

I want my clone to use double strafe on sniper clone.

### How to Reproduce

I Just Trying to splay with clone skill, 
The clone used skill but it does not use AC_DOUBLE or 

### Official Information

Rathena #5931

**Comment by mplaten** [comment]
rathena hash seens incorrect...

**Comment by mplaten** [comment]
Okay, no meme... Clones can use skills? At last rathena, clone are spawned with MD_NOCAST...

**Comment by krispaul27** [comment]
> Okay, no meme... Clones can use skills? At last rathena, clone are spawned with MD_NOCAST...

Yeah Clones Use's Skills. You should edit your mob_clone_spawn () it works. Anyway i just want to find out how clone uses skills.
kindly post the code so i can study it.
